packages/gatsby/src/db/loki/nodes-query.js

```diff
@@ -180,9 +204,34 @@ const fixNeTrue = filter =>
     return acc
   }, {})
 
+const liftResolvedFields = (args, resolvedFields) => {
```

Comment:
Same as above: Looks like this is the same in `redux/run-sift` and `loki/nodes-query`? can we move it do `src/db/common/`?